backend/src/services/__tests__/player.service.test.ts

```diff
@@ -34,11 +35,16 @@ describe("createPlayer Service", () => {
   it("creates a player and returns its playerId", async () => {
     const gameCode = await createGame();
 
-    const playerId = await createPlayer(gameCode, "Bob");
+    const { playerId, token } = await createPlayer(gameCode, "Bob");
 
     const game = await getGame(gameCode);
 
-    expect(game.players.id(playerId)).not.toBeNull();
+    const player = game.players.id(playerId);
+    expect(player).toBeDefined();
+    if (player === null) return;
```

Comment:
doesn't return pass the test? Is this a valid state for player, or should it run fail()?

Reply:
Yep good catch!